### ISSUE TITLE
wilder causing `galaxyline` to fill with `^`

### ISSUE THREAD

**Issue by ExpandingMan** [gelguy/wilder.nvim#153]
Wilder is causing visual artifacts to appear in [galaxyline](https://github.com/glepnir/galaxyline.nvim) outside of where wilder is supposed to render.

![image](https://user-images.githubusercontent.com/11379353/185260610-28b66715-52d2-475e-b1da-8cbe0f1901cc.png)

I haven't yet been able to reproduce this without galaxy line but I am certain the problem is being caused by wilder since it disappears when I disable wilder and only shows up when wilder is active.

I've tried stripping down my wilder config, but it's already pretty minimal and so far I have a problem with the `^` artifacts whenever I can get wilder to render anything.  My current config can be found [here](https://gitlab.com/ExpandingMan/neovimconfig/-/blob/master/lua/etc/wilder.lua)

This occurs both on latest neovim tagged and latest neovim nightly.

**Comment by ExpandingMan** [comment]
I have tracked this down to something very strange, when I set
```lua
vim.api.nvim_set_hl(0, "StatusLineNC", {fg=colors.bgdark, bg=colors.bgdark})
```
this problem shows up as described above, but when removed it disappears.  This effectively solves the issue for me as of now since I don't have a need for `StatusLineNC` (or even know what the hell it is, frankly).

Not sure if this is a bug in wilder, galaxyline, nvim itself or what.
